koupleless-ext/koupleless-ext-web-gateway/src/test/resources/biz1_forwards.yaml

```
@@ -0,0 +1,5 @@
+- contextPath: test1
+  hosts:
+    - test1
+  paths:
+    - from: /test1
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)